lib/daru/io/importers/linkages/sql.rb

```diff
@@ -0,0 +1,27 @@
+require 'daru'
+
+module Daru
+  class DataFrame
+    class << self
+      # Read a database query and returns a Dataset
+      #
+      # @param dbh [DBI::DatabaseHandle, String] A DBI connection OR Path to a SQlite3 database.
+      # @param query [String] The query to be executed
+      #
+      # @return A dataframe containing the data resulting from the query
+      #
+      # USE:
+      #
+      #  dbh = DBI.connect("DBI:Mysql:database:localhost", "user", "password")
+      #  Daru::DataFrame.from_sql(dbh, "SELECT * FROM test")
+      #
+      #  #Alternatively
+      #
+      #  require 'dbi'
+      #  Daru::DataFrame.from_sql("path/to/sqlite.db", "SELECT * FROM test")
+      def from_sql(dbh, query)
+        Daru::IO::Importers::SQL.load dbh, query
+      end
+    end
+  end
+end
```

Comment:
I am not sure I like this "linkages" naming and structure. In fact, it is better for `daru-io` to introduce as small changes to daru core classes as possible. It is easy to achieve this way:
* define module with appropriate method in `Daru::IO` namespace
* then `Daru::Dataframe.extend Daru::IO::Importers::SQL::Extension` (or something like that)

Another way (probably should postpone it for later) is use a bit of metaprogramming, and write something like:
```ruby
Daru::Dataframe.register_importer :from_sql, Daru::Importers::SQL
```

In both of cases, the _main_ docs of importer/exporter (which arguments it expects, how it works and what it requires) should go to the main importer/exporter class or module, not to method suddenly dropped into the main library.